app/src/main/java/com/example/themoviesdb/ui/MovieDetailsFragment.kt

```diff
@@ -0,0 +1,52 @@
+package com.example.themoviesdb.ui
+
+import android.os.Bundle
+import android.view.*
+import androidx.databinding.DataBindingUtil
+import androidx.fragment.app.Fragment
+import com.bumptech.glide.Glide
+import com.example.themoviesdb.R
+import com.example.themoviesdb.databinding.FragmentMovieDetailsBinding
+import com.example.themoviesdb.domain.model.Movie
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+class MovieDetailsFragment : Fragment() {
+
+    private lateinit var binding: FragmentMovieDetailsBinding
+    private var movie: Movie? = null
```

Comment:
fragments/activities shouldn't deal with models, they should communicate with the viewModel.
'movie' variable should be in the viewModel.